framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1408,6 +1441,20 @@ public static Range getRange(AnnotationMirror rangeAnno) {
     }
 
     /**
+     * Returns a {@code Range} bounded by the values specified in the given {@code @IntRange} or
+     * {@code @IntVal} annotation. Returns null if the given annotation is not {@code IntRange} or
+     * {@code IntVal}.
+     */
+    public static Range getRangeOrConvertIntVal(AnnotationMirror anno) {
```

Comment:
I think this method should be folded into `getRange`.  Then, clients don't have to think about which of the two to call.  The two have essentially identical specifications.  That will also address the problem that this one's name says too much about the implementation.